cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/GcpPlatformResources.java

```diff
@@ -184,18 +192,25 @@ public CloudNetworks networks(CloudCredential cloudCredential, Region region, Ma
         return new CloudNetworks(result);
     }
 
-    public SubnetworkList getSubnetworkList(Region region, Compute compute, String projectId, String subnetId, String sharedProjectId) throws IOException {
+    public SubnetworkList getSubnetworkList(Region region, Compute compute, String projectId, List<String> subnetIds,
```

Comment:
this could be a private method and we could skip null check on `subnetIds` if the my previous comment is addressed